File: cs/Backend/Api/Models/PostDto.cs
using System;
using Newtonsoft.Json;
using Socneto.Domain.Models;

namespace Socneto.Api.Models
{
    public class PostDto
    {
        public string AuthorId { get; set; }
        
        public string Text { get; set; }

        public DateTime PostedAt { get; set; }

        public static PostDto FromValue(Post post)
        {
            return new PostDto
            {
                AuthorId = post.AuthorId,
                Text = post.Text,
                PostedAt = post.PostedAt,
            };
        }
    }
}

File: storage/analysis-storage/src/main/java/cz/cuni/mff/socneto/storage/analysis/results/api/dto/PostDto.java
package cz.cuni.mff.socneto.storage.analysis.results.api.dto;

import lombok.Builder;
import lombok.Data;


@Data
@Builder
public class PostDto {
    private String id;
    private String text;
    private String authorId;
    private String source;
    private String dateTime;
}
